pkg/config/config_policy_routing.go

```diff
@@ -89,7 +91,9 @@ var PolicyRoutingConfigSet = Set{
 	nil,
 }
 
-func init() {
+// InitPolicyRouting performs necessary initialization for policy routing.
+// It should be called before running the policy routing enforcement loop.
+func InitPolicyRouting() {
```

Comment:
Since it's no longer the special init() function, it can return an error. I feel it's better to return the error all the way and Fatal() in main(). Alternatively, the functions in this calling chain should be called MustXXX() to indicate that it can crash the whole program (while I prefer returning errors).